bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/sequencefiles/HadoopSerializationCoder.java

```diff
@@ -70,15 +69,6 @@ private void readObject(java.io.ObjectInputStream in) throws IOException, ClassN
     }
   }
 
-  /**
-   * Disable backwards compatibility with previous versions that were serialized
-   * @throws InvalidObjectException
-   */
-  private void readObjectNoData() throws InvalidObjectException {
-    throw new InvalidObjectException("Stream data required");
-  }
-
-
```

Comment:
why remove this?

Reply:
I thought it is not used, no?

Reply:
It's implicitly used for java serialization:
https://docs.oracle.com/javase/7/docs/platform/serialization/spec/input.html#6053

Reply:
Good catch! I was having doubt about this one. I should have pointed out earlier.
I added some comment to remind us not to delete this method.